content/general/push/activate-subscribe.textile

```diff
@@ -15,9 +15,11 @@ jump_to:
     - Subscribing to push notifications#subscribing
 ---
 
-<%= partial partial_version('general/push/_activate_subscribe_intro') %>
+Every device that will receive push notifications must activate itself with the local operating system or framework, and hook into the push notification services that the underlying platform provides. This functionality is platform-specific and can also vary considerably across not just platforms, but also across the push services that operate on those platforms such as GCM and FCM, both of which are available on the Android platform.
 
-The client libraries also provide a set of admin functionality that is typically used server-side (in a trusted environment) to manage all devices and push notification delivery and subscriptions. You can find out more in the "push admin documentation":admin.
+The Ably client libraries aim to abstract away this complexity and platform-specific behaviour by providing a consistent API for device activation, maintenance of the device registration, subscription to Ably channels for receiving push notifications, and of course for receiving the native push notifications themselves.
```

Comment:
"and of course for receiving the native push notifications themselves": we don't do this bit